reconcile/utils/mr/promote_qontract.py

```
@@ -206,6 +215,57 @@ def _process_file_with_json_paths(
+
+        This must not raise: unlike the other process() steps, it depends
+        on the OPA image already being mirrored to quay.io, which can lag
+        behind this MR by a few minutes. Any failure here would otherwise
+        abort the whole MR (see MergeRequestBase.submit_to_gitlab) and
+        block the unrelated bumps below. On failure we just skip - the
+        existing pin stays in place and we retry on the next promotion.
+        """
+        image_uri = f"{OPA_MASTER_IMAGE}:{self.commit_sha}"
+        try:
+            img = Image(image_uri, timeout=IMAGE_REQUEST_TIMEOUT)
+        except Exception as e:
```

Ruff (0.15.21)

[warning] Do not catch blind exception: `Exception`

(BLE001)